decidim-core/config/locales/en.yml

```diff
@@ -176,6 +176,10 @@ en:
       nesting_too_deep: can't be inside of a subcategory
   layouts:
     decidim:
+      cookie_warning:
+        description_html: This site uses cookies. By continuing to browse the site, you agreeing to our use of cookies. Find out more about it %{link}.
```

Comment:
Previous version I wasn't sure, this one is plain wrong (you are missing the `are` after the subject for the continuous tense)

`you agree to` is the right way